api/prisma/migrations/20240213151159_link_to_version_pre_script/migration.sql

```diff
@@ -0,0 +1,38 @@
+/*
```

Comment:
Not sure what tool you use for migrations, this looks a good candidate to wrap in a transaction if the tool doesn't already

Reply:
These are generated with prisma migrate. From reading about it it doesn't stop you using transactions (or wrap them automatically), you just have to put BEGIN and COMMIT in - I'll add this

Reply:
I've now pushed this up - it involved some reverting and cherry picking back of commits because we need a code state to checkout that has only the first migration (but in its transaction-using state) and the script